packages/core/src/generators/builder/generator.ts

```diff
@@ -186,6 +186,57 @@ const componentMappers: {
     );
   },
   Show(node, options) {
+    const elseCase = node.meta.else as MitosisNode;
+    const children = node.children.filter(filterEmptyTextNodes);
+    const showNode =
+      children.length > 0
+        ? el(
+            {
+              // TODO: the reverse mapping for this
+              component: {
+                name: 'Core:Fragment',
+              },
+              bindings: {
+                show: node.bindings.when?.code as string,
+              },
+              children: children.map((node) => blockToBuilder(node, options)),
+            },
+            options,
+          )
+        : undefined;
+
+    const elseNode =
+      elseCase && filterEmptyTextNodes(elseCase)
+        ? el(
+            {
+              // TODO: the reverse mapping for this
+              component: {
+                name: 'Core:Fragment',
+              },
+              bindings: {
+                show: `!(${node.bindings.when?.code as string})`,
```

Comment:
this alternatively can just use `hide`
```suggestion
                hide: node.bindings.when?.code as string,
```